modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+		}
+	  });
+	}
+----
+
+NOTE: The comment sidebar presents with no animation or other effect. It is just there, along with any other initialised {productname} editor components. Also, as noted above, the sidebar does not take focus, which remains with the main editor.
+
+
+==== Support annotating certain block elements directly when using the editor annotator API
+
+Several new block elements can now have comments attached to them, including:
+
+* Tables of contents
+* Images
+* Images with captions
+* Code samples
+* media in iframes
+* embedded media
+* embedded pages
+
+When comments are showing, a 3px wide yellow (hex color #FFE89D) rectangle presents around any element with a comment attached.
+
+NOTE: Enablng this feature depends on changes made to the Core {productname} editor. As a consequence, adding comments to these elements requires both {productname} 6.1 or later and version 3.1.0 or later of the xref:comments.adoc[Comments Premium Plugin].
+
+
+==== The label name for the Save Comment button can now be customized
+
+Previously, the Save Comment was labeled *Save*. After field reports that end-users were interpreting this to mean the entire {productname} document would be saved on pressing this button, two changes have been made.
+
+1. The default label for this button is now *Comment*.
++
+This new default presents for both new comments and replies to existing comments.
+
+2. This default label is now customisable.
++
+_How does one customise this? I presume it’s a value one sets as part of `tinymce.init` but which one? And where in this configuration file?_
++
+_Or is it a matter of an integrator editing the `.pot` file — `src/text/pot/plugins/tinycomments/tinymce_tinycomments.pot` — that comes along for the ride?_
+
+NOTE: when a comment is being edited, the default button, which writes the changes back and closes the Edit Comment dialog, is still labeled *Save*.
+
+
+For information on the **Comments** plugin, see: xref:comments.adoc[Comments plugin].
+
+
+=== Enhanced Media Embed 3.0.1
+
+The {productname} 6.1 release includes an accompanying release of the **Enhanced Media Embed** premium plugin.
+
+**Enhanced Media Embed** 3.0.1 fixes the following bug:
+
+==== Enhanced Media Embed tests failed in several browsers
+
+An incorrect `RealKeys()` setting caused the Enhanced Media Embed webdriver tests to fail in several browsers (including Firefox, Microsoft Edge, and Safari).
+
+With this update, this setting has been corrected, from `RealKeys.text(\n)` to `RealKeys.combo({}, 'enter')`. As a consequence, the Enhanced Media Embed webdriver tests now pass in all supported browsers, as expected.
+
+For information on the **Enhanced Media Embed** plugin, see: xref:enhancedmediaembed.adoc[Enhanced Media Embed plugin].
+
+
+=== Page Embed 2.1.0
+
+The {productname} 6.1 release includes an accompanying release of the **Page Embed** premium plugin.
+
+**Page Embed** 2.1.0 includes the following bug fix:
```

Comment:
This section isn't correct... it never supported translations fully so this was an improvement to add the translations not a bug fix.

Reply:
I feel almost recalcitrant arguing the point, but [TINY-8495](https://ephocks.atlassian.net/browse/TINY-8495) presents this pretty straightforwardly as a bug:

> @ben.tran  nice find mate!
> 
> To confirm, we are missing these 3 strings for the 5 langs noted, I will followup with Lionbridge
> 
> Long description URL
> Show iframe border
> Enable scrollbar

And your comment adds to the ‘this was a bug by omission’ sense:

> please be sure to setup PageEmbed properly for translations as it’s never been properly translated.

At this point we’re discussing semantics. Which is a good thing to discuss in this particular regard. But I’m somewhat happy with the notion that, from the end-user and even integrator’s perspective, this is a correction (even if only by originally accidental omission) rather than an improvement.

Reply:
The linked Jira is an improvement and it's already been implemented as such, so it needs to stay that way I'm afraid. The existing translations as I understand it, only worked because the strings were already translated elsewhere.

Reply:
If you want to document it this way still though, I'm not going to argue, it's just it'll be out of sync with the items shipped in the 6.1 release 🤷

Reply:
Re-wrote this section to present the change as an enhancement rather than as a bug fix.